### ISSUE TITLE
Cast is not valid on Embedded texture

### ISSUE THREAD

**Issue by ZorroMundoYT** [krzys-h/UndertaleModTool#1107]
### Describe the bug

The bug is about Embedded textures, which (I dobut) that is implemented on UndertaleModTool.

### Reproducing steps

Open "Depper Jar" and then it crashes.

### Setup Details

UndertaleModTool compilation 6501d9a
Windows 10 22h2 19045.2130
GameMaker runtime 2022.9.1.51

**Comment by Dobby233Liu** [comment]
Can you show a screenshot of the error message?

**Comment by ZorroMundoYT** [comment]
![image](https://user-images.githubusercontent.com/50798713/200099833-1d14a908-ccd3-4cfe-ad7b-dd0138bc1ed9.png)
![image](https://user-images.githubusercontent.com/50798713/200099847-82762036-f17f-411a-a1fd-ff0e43665491.png)

**Comment by QuantumNoisemaker** [comment]
Ran into this issue as well, same error messages as ZorroMundoYT. I attempted to extract the archive with an older version, but then ran into the same error with the most up to date bleeding edge build.

Reproducing steps
Open "Secret Care Cafe" and then it crashes.

Setup Details
UndertaleModTool compilation https://github.com/krzys-h/UndertaleModTool/commit/6501d9ad79c2ad80b6282882e17692a534b9dea2
Windows 10 Pro 21H1 Version 10.0.19043 Build 19043


![Screenshot 2022-11-05 052444](https://user-images.githubusercontent.com/3668
![Screenshot 2022-11-05 052458](https://user-images.githubusercontent.com/36684562/200113059-fd53aca0-850d-415d-a571-8352e8849a8f.png)
4562/200113053-5006ad7d-f17d-44cb-9d87-fbfdf41caef3.png)
